src/components/pages/Explore/Explore.tsx

```diff
@@ -47,87 +131,42 @@ export default function Explore({initialFilters}: ExploreProps) {
-                            <div style={{width: "80%"}}>
-                                {Translate("UnityBanner", ["Currently"])}
-                              <a href={"https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/early-investigations"}
-                                 target="_blank" rel="noopener noreferrer">
-                                {Translate("UnityBanner", ["ProtocolLink"])}
-                              </a> {Translate("UnityBanner",["ViewDatabase"])} <span className={"link"} onClick={() => {dispatch({type: "SET_UNITY_FILTER_PULSATE", payload: true})}}> {Translate("UnityBanner", ["WHOUnityFilter"])} </span>
-                                {Translate("UnityBanner", ["Filter"])}
-                            </div>
-                            <Icon link onClick={() => {setShowUnityBanner(false)}} name='close'/>
-                    </div>)}
-                        <div className="flex w-100">
+            {renderMobileModal(mobileModalOpen, setMobileModalOpen)}
+            <div className="explore__dashboard">
+                {!isMobileDeviceOrTablet ? (
+                    <>
+                        {showUnityBanner && renderUnityBanner(dispatch, setShowUnityBanner)}
+                        <div className="flex w-100 explore__container">
                             <div className="col-2 p-0 flex">
-                                <LeftSidebar page={PageStateEnum.explore}/>
+                                <LeftSidebar page={PageStateEnum.explore} />
                             </div>
                             <div className="col-8 p-0 flex" id={PAGE_HASHES.Explore.Map}>
-                                <Loader indeterminate active={state.explore.isLoading}/>
+                                <Loader indeterminate active={state.explore.isLoading} />
                                 <div className="info flex legend center-item">
-                                    <Legend/>
+                                    <Legend />
                                 </div>
                                 <MapboxMap
                                     countriesConfig={{
-                                        estimateGradePrevalences: state.explore.estimateGradePrevalences
+                                        estimateGradePrevalences: state.explore.estimateGradePrevalences,
                                     }}
                                     studyPinsConfig={{
-                                        records: state.explore.records
+                                        records: state.explore.records,
                                     }}
                                 />
                             </div>
                             <div className="col-2 p-0 flex sidebar-container">
-                                <Filters page={PageStateEnum.explore}/>
+                                <Filters page={PageStateEnum.explore} />
```

Comment:
Why not put this whole block of code in a seperate function similar to <ExploreMbile> for consistency purposes and improved readbility?

Reply:
I don't think this block of code is as hard to read as before, because its been simplified to its main functional components, along with the divs for simple structuring. I think simplifying it more wouldn't be that useful

Reply:
Makes sense!